### ISSUE TITLE
NoClassDefFoundError for AdaptiveVideoTrackSelection$Factory

### ISSUE THREAD

**Issue by langrenfengzi** [eneim/Toro#190]
When using ExoPlayerView in "im.ene.toro2:toro-extended:2.2.0", get this error:
```
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/exoplayer2/trackselection/AdaptiveVideoTrackSelection$Factory;
 at im.ene.toro.exoplayer2.ExoPlayerView.initializePlayer(ExoPlayerView.java:195)
 at im.ene.toro.exoplayer2.ExoPlayerView.setMediaSource(ExoPlayerView.java:114)
 at im.ene.toro.exoplayer2.ExoPlayerView.setMedia(ExoPlayerView.java:104)
 at im.ene.toro.exoplayer2.ExoPlayerView.setMedia(ExoPlayerView.java:96)
 ...
```
And I found this, it was committed  on Mar 16   ╮（╯＿╰）╭
[Rename AdaptiveVideoTrackSelection to AdaptiveTrackSelection](https://github.com/google/ExoPlayer/commit/ce5c0c18f952caa19b863345252fd267804d7e83)

**Comment by eneim** [comment]
@langrenfengzi Do you have manually add ExoPlayer to your dependencies? In latest one it has changed the name of that class ...

**Comment by langrenfengzi** [comment]
Yes, I encountered this problem when used the latest exoplayer library r2.3.1.
Use the version  r2.2.0 and below should be ok.

**Comment by eneim** [comment]
I'm trying to add the support in next release. Please wait ...

**Comment by eneim** [comment]
Fixed in 2.3.0. Note: always use latest ExoPlayer.
